docs/docs/command-line-interface.mdx

```diff
@@ -585,6 +586,25 @@ key is missing or if the default value has not been changed.
 If you pass a `max_history` value to one or more policies in your `config.yml` file, provide the
 smallest of those values in the validator command using the `--max-history <max_history>` flag.
 
+### Validate flows
+
+The validator will perform the following checks on flows:
+- determine whether flow names or descriptions are unique after stripping punctuation
+- verify whether natural language conditions used in branch steps or `collect` step rejections are valid `pypred` expressions
+- determine whether slots used in flows are defined in the domain
+- disallow `list` slots from being used in flows
```

Comment:
maybe we want to give more context on this choice?

Reply:
The only context I have is that Studio doesn't support `list` slots, but don't know the deeper reason; @dakshvar22 do you have more context here?

Reply:
Added more context here: https://github.com/RasaHQ/rasa/pull/12916/commits/7d2c6742c58cdec959211ec65b609b85c0cc3d87